src/main/scala/com/cognite/sdk/scala/common/sttps/GzipSttpBackend.scala

```diff
@@ -1,17 +1,18 @@
 // Copyright 2020 Cognite AS
 // SPDX-License-Identifier: Apache-2.0
 
-package com.cognite.sdk.scala.common
+package com.cognite.sdk.scala.common.sttps
 
-import java.io.ByteArrayOutputStream
-import java.nio.charset.Charset
-import java.util.zip.GZIPOutputStream
-import sttp.client3._
 import org.apache.commons.io.IOUtils
 import sttp.capabilities.Effect
+import sttp.client3._
 import sttp.model.{Header, HeaderNames}
 import sttp.monad.MonadError
 
+import java.io.ByteArrayOutputStream
+import java.nio.charset.Charset
+import java.util.zip.GZIPOutputStream
+
 class GzipSttpBackend[F[_], +P](delegate: SttpBackend[F, P], val minimumSize: Int = 1000)
```

Comment:
Since we're moving this file to a package that already includes `sttp`, let's rename it to `GzipBackend` so it has the same naming convention as the other classes in this package.